karma.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
Easy automation, works well with multiple browsers, code watch works well, good tools integration and support. Pretty common in the wild.